static/src/js/middleware/metrics/__tests__/index.test.js

```diff
@@ -132,6 +133,22 @@ describe('metrics middleware', () => {
     expect(events.spatialEdit).toHaveBeenCalledWith(action)
   })
 
+  test('calls spatialSelection event', () => {
+    // eslint-disable-next-line no-import-assign
```

Comment:
You can remove all of the `eslint-disable`s in this file with this mock at the top of the file
```
jest.mock('../events', () => ({
  collectionSortChange: jest.fn(),
  dataAccess: jest.fn(),
  defaultClick: jest.fn(),
  map: jest.fn(),
  relatedCollection: jest.fn(),
  spatialEdit: jest.fn(),
  spatialSelection: jest.fn(),
  timeline: jest.fn(),
  timing: jest.fn(),
  virtualPageview: jest.fn()
}))
```

Reply:
Great call out thanks